jedi.inc

```diff
@@ -1590,6 +1590,10 @@
     {$DEFINE SUPPORTS_OUTPARAMS}
     {$DEFINE SUPPORTS_WIDECHAR}
     {$DEFINE SUPPORTS_WIDESTRING}
+    {$IFDEF CPU64}
+		{$DEFINE USE_64BIT_TYPES}
```

Comment:
What's the point of this define? It's not even used by your other pull request, and I don't see how a 64 bits CPU forces you to use 64bits types everywhere. This is not true for x86_64 and may not even be true for other CPU types.

Reply:
May I ask what the purpose of the USE_64BIT_TYPES define is? It's only used further down in COMPILER16_UP, but it's not documented inside the comments at the top. 🤔

Reply:
- if you agree, I can completely remove the `USE_64BIT_TYPES` from `jedi.inc` and leave the header of the function mentioned above as `function InternalReadProcMem(ProcessHandle: THandle; Address: DWORD; Buffer: Pointer; Size: SIZE_T): Boolean;`, using the `Size: SIZE_T` parameter;

- haven't checked whether `USE_64BIT_TYPES` is used in other packages of the JEDI project;

- there are several pull requests that are on hold, but separated them like this at the advice given from people on the newsgroup. They were saying that there should be separate pull requests for each bug fixed, as well as for having FPC compile JCL for both 32-bit and 64-bit. This is the reason for which at the moment `USE_64BIT_TYPES` doesn't seem to be used by any other pull requests. Those pull requests would be on their way, but as mentioned above, this conditional could be removed if it is not used in other JEDI packages.